pkg/base1/cockpit.js

```diff
@@ -2977,7 +3013,7 @@ function full_scope(cockpit, $, po) {
             return low;
         }
 
-        self.load = function load(beg, end) {
+        self.load = function load(beg, end, for_walking) {
```

Comment:
Hmm, the fact that beg, end is always present means that people can simply ignore for_walking and the right thing will happen.  Your cadvisor polling code will not need any change, for example.